servers/src/test/java/tachyon/worker/block/evictor/LRFUEvictorTest.java

```diff
@@ -0,0 +1,297 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block.evictor;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.worker.block.BlockMetadataManager;
+import tachyon.worker.block.BlockMetadataManagerView;
+import tachyon.worker.block.BlockStoreEventListener;
+import tachyon.worker.block.BlockStoreLocation;
+import tachyon.worker.block.meta.StorageDir;
+
+/**
+ * Sanity check on specific behavior of LRFUEvictor such as evicting/moving least blocks with
+ * minimum CRF value and cascading LRFU eviction.
+ */
+public class LRFUEvictorTest {
+  private static final long USER_ID = 2;
+  private static final long BLOCK_ID = 10;
+
+  private BlockMetadataManager mMetaManager;
+  private BlockMetadataManagerView mManagerView;
+  private Evictor mEvictor;
+
+  private double mStepFactor;
+  private double mAttenuationFactor;
+
+  @Rule
+  public TemporaryFolder mTestFolder = new TemporaryFolder();
+
+  @Before
+  public final void before() throws Exception {
+    File tempFolder = mTestFolder.newFolder();
+    mMetaManager = EvictorTestUtils.defaultMetadataManager(tempFolder.getAbsolutePath());
+    mManagerView =
+        new BlockMetadataManagerView(mMetaManager, Collections.<Integer>emptySet(),
+            Collections.<Long>emptySet());
+    TachyonConf conf = new TachyonConf();
+    conf.set(Constants.WORKER_EVICT_STRATEGY_CLASS, LRFUEvictor.class.getName());
+    mStepFactor = conf.getDouble(Constants.WORKER_EVICT_STRATEGY_LRFU_STEP_FACTOR, 0.25);
+    mAttenuationFactor =
+        conf.getDouble(Constants.WORKER_EVICT_STRATEGY_LRFU_ATTENUATION_FACTOR, 2.0);
+    mEvictor = Evictor.Factory.createEvictor(conf, mManagerView);
```

Comment:
Don't you need to include `Evictor.java` (and the change to `Evictor.Factory.createEvictor`) in this PR, in order to make this test passed?